src/plugins/intel_cpu/src/utils/bfloat16.hpp

```diff
@@ -22,7 +22,7 @@ namespace ov::intel_cpu {
 class bfloat16_t {
 public:
     bfloat16_t() = default;
-    bfloat16_t(float value) noexcept
+    bfloat16_t(float value) noexcept  // NOLINT(google-explicit-constructor)
```

Comment:
Please add a reason.
In how many places we will have to fix it if we remove NOLINT?

Reply:
Huge diff, increased code verbosity